src/mode.js

```diff
@@ -173,5 +173,10 @@ export function getFullVersion_(win, isLocalDev) {
     return win.AMP_CONFIG.v;
   }
 
-  return version;
+  // Currently `$internalRuntimeVersion$` and thus `mode.version` contain only
+  // major version. The full version however must also carry the minor version.
+  // We will default to production default `01` minor version for now.
+  // TODO(erwinmombay): decide whether $internalRuntimeVersion$ should contain
+  // minor version.
+  return `01${version}`;
```

Comment:
Yes, `01` is not always the best. But it should always be there. And we have no better information for a fallback at this time. Separately, I asked @erwinmombay to consider to simply update `$internalRuntimeVersion$` to a full version or add another replacement like this for minor version.